examples/shader/fallback_image.rs

```diff
@@ -0,0 +1,72 @@
+use bevy::{
```

Comment:
This needs module-level doc strings explaining the purpose of the example.

Reply:
I'll just move the docs I wrote for the `main` function. I just copied from another example.